### ISSUE TITLE
blackscreen

### ISSUE THREAD

**Issue by ghost** [Luna5ama/TrollHack#27]
Whenever I start it I get a blackscreen, idk how to solve it

**Comment by TwoThreeTwo** [comment]
send latest.log 


windows key + r
%appdata%
click .minecraft
go to logs
find 'latest.log'
post it in this chat

luna will evetually message you would a solution

**Comment by Zane2b2t** [comment]
[latest.log](https://github.com/Luna5ama/TrollHack/files/9313305/latest.log)
I have the same issue, no longer crashes like with 0.3 but on 0.4 its black screen
@Luna5ama

**Comment by Luna5ama** [comment]
```
[03:33:49] [Client thread/INFO]: [me.earth.earthhack.impl.util.render.GlShader:createShader:259]: Shader chams could not be linked: WARNING: 0:33: 'assing' : implict conversion between types allowed from GLSL 1.20 
WARNING: 0:60: 'assing' : implict conversion between types allowed from GLSL 1.20 
WARNING: 0:61: 'assing' : implict conversion between types allowed from GLSL 1.20 
WARNING: 0:63: 'assing' : implict conversion between types allowed from GLSL 1.20 
ERROR: 0:109: '=' :  cannot convert from '4-component vector of float' to '2-component vector of float'
```
Looks likes an earthhack issue, your gpus supports GL 3.1 so TrollHack things *should* be fine except FastRender.

**Comment by Zane2b2t** [comment]
> 

you should try making it compatiable

**Comment by TwoThreeTwo** [comment]
If it's an earthhack issue then 3arth devs have to fix it

**Comment by Zane2b2t** [comment]
> If it's an earthhack issue then 3arth devs have to fix it

3arthqu4ke said he will u update soon :tm:

**Comment by Zane2b2t** [comment]
> If it's an earthhack issue then 3arth devs have to fix it

i think i know the problem, phobos has costum icons for the main menu i think its conflicting  with trollhack's REALY costum main menu

**Comment by 3arthqu4ke** [comment]
I dont have this issue. Both clients work together ~~fine~~

**Comment by Luna5ama** [comment]
They are using Intel iGPU and the driver doesn't allow "illegal" stuff in the shader, which Nvidia and AMD (probably) drivers allowed.

**Comment by 3arthqu4ke** [comment]
But is it your main menu shader or my shader? The one from 3arthh4ck you see in the log fails to link but would not get used until an entity is rendered, thats whats confusing me. 

Maybe @Zane2b2t can try Trollhack standalone?

**Comment by Zane2b2t** [comment]
> But is it your main menu shader or my shader? The one from 3arthh4ck you see in the log fails to link but would not get used until an entity is rendered, thats whats confusing me.
> 
> Maybe @Zane2b2t can try Trollhack standalone?

Trollhack worked standlone but sometimes it crashes suddenly

**Comment by Luna5ama** [comment]
> > But is it your main menu shader or my shader? The one from 3arthh4ck you see in the log fails to link but would not get used until an entity is rendered, thats whats confusing me.
> > Maybe @Zane2b2t can try Trollhack standalone?
> 
> Trollhack worked standlone but sometimes it crashes suddenly

The crash should be fixed already in 0.0.5
